src/Aspire.Hosting.AWS/Utils/Internal/ProcessCommandService.cs

```diff
@@ -98,4 +109,62 @@ internal class ProcessCommandService : IProcessCommandService
         return new IProcessCommandService.RunProcessAndCaptureStdOutResult(process.ExitCode, output.ToString());
 
     }
+    
+    /// <summary>
+    /// Utility method for running a command on the commandline. It returns backs the exit code.
+    /// </summary>
+    /// <param name="logger"></param>
+    /// <param name="path"></param>
+    /// <param name="arguments"></param>
+    /// <param name="workingDirectory"></param>
+    /// <returns></returns>
+    public int RunProcess(ILogger logger, string path, string arguments, string workingDirectory)
```

Comment:
Was there a reason the existing `RunProcessAndCaptureOuputAsync` didn't work? Looks like they do basically the same thing except the original also returned back the output. If you used that one then LambdaLifecycleHook that calls this we could check the error code and if it was non-Zero log the output and since in that context we know it is fatal use `LogError`?

Reply:
`RunProcessAndCaptureOuputAsync` is an async function while the new one isn't. I tried using the existing one but there's a deadlock happening somewhere. I ran 5 runs and they all timed out after 2 hours.

Reply:
I'm fine for now since these are internal but eventually we should figure out how to consolidate this.